src/helpers/figures/charts/bar_chart.ts

```diff
@@ -231,7 +232,7 @@ export function createBarChartRuntime(chart: BarChart, getters: Getters): BarCha
       datasets: getBarChartDatasets(definition, chartData),
     },
     options: {
-      ...CHART_COMMON_OPTIONS,
+      ...getChartDefaultOptions("bar"),
       indexAxis: chart.horizontal ? "y" : "x",
       layout: getChartLayout(definition, chartData),
       scales: getBarChartScales(definition, chartData),
```

Comment:
add `onHover: getBarChartOnHover(...)` methods to stay consistend with the rest, rather than a `getChartDefaultOptions`